python/lsst/ap/association/diaPipe.py

```diff
@@ -451,11 +451,138 @@ def run(self,
         # Accept either legacySolarSystemTable or optional solarSystemObjectTable.
         if legacySolarSystemTable is not None and solarSystemObjectTable is None:
             solarSystemObjectTable = legacySolarSystemTable
+
```

Comment:
Not your change, but `solarSystemObjectTable` should be added to the docstring above (I can't comment on unchanged lines, so I had to put it here). You can just copy what you have in `associateDiaSources`